install-panel.sh

```diff
@@ -117,11 +142,15 @@ function print_brake {
     echo ""
 }
 
-hyperlink() {
+function hyperlink() {
   echo -e "\e]8;;${1}\a${1}\e]8;;\a"
 }
 
-required_input() {
+#################################
+##### User input functions ######
+#################################
+
+function required_input() {
```

Comment:
Then I would like to ask where else it should be defined? What the category should be called? I think I grouped them pretty good, because this section also has  `ask_firewall` and `ask_letsencrypt`.